src/components/Dashboard/TalkDashboard.vue

```diff
@@ -73,6 +73,8 @@ async function actualiseData() {
 onMounted(() => {
 	actualiseData()
 	actualiseDataInterval = setInterval(actualiseData, 300_000)
+
+	EventBus.on('refresh-talk-dashboard', actualiseData)
 })
```

Comment:
I'd restart a timer here in that case. Also could be called earlier without onMounted hook?
```suggestion
initActualizeData()
EventBus.on('refresh-talk-dashboard', initActualizeData)

function initActualizeData() {
	if (actualiseDataInterval) {
		clearInterval(actualiseDataInterval)
	}
	actualiseData()
	actualiseDataInterval = setInterval(actualiseData, 300_000)
}
```